src/main.dart

```diff
@@ -1,8 +1,118 @@
-// TODO: Run task list app
-void main(List<String> args) {
-  print('Hello World');
+import 'dart:io';
+
+class Task {
+  int taskID;
+  String taskName;
+  String description;
+  bool isCompleted;
+  
+  Task(this.taskID, this.taskName, this.description, this.isCompleted);
 }
 
-class TaskListApp {
-  // TODO: Implement Task list app
+ class Funcions {
+
+List<Task> tasks = [];
+
+int taskID = 1;
+//addTask
+ void addTask(String taskName, String description) {
+tasks.add(Task(taskID,taskName,description, false));
+taskID++;
```

Comment:
It's a good practice to provide feedback to the user when they perform actions, like adding, completing, or removing tasks.

print('Task added.');